seb_openedx/permissions.py

```diff
@@ -54,13 +55,17 @@ def get_seb_keys(self, course_key):
                 ]
             }
         """
+        all_keys = []
         for source_function in get_ordered_seb_keys_sources():
             seb_keys = source_function(course_key)
             if isinstance(seb_keys, dict):
                 seb_keys = seb_keys.get(self.detailed_config_key, None)
-            if seb_keys:
+            if seb_keys and settings.SEB_USE_ALL_SOURCES:
+                all_keys += seb_keys
```

Comment:
Should we delete repeated keys if they exist? Does it make sense to use a set?

Reply:
Good idea. I added this